services/web-server/src/graphql/WorkerManager.graphql

```diff
@@ -1,19 +1,19 @@
 type WorkerManagerWorkerPoolSummary {
-    workerPool: String!
+    workerPoolId: String!
     description: String
     owner: String!
-    minCapacity: Int!
-    maxCapacity: Int!
-    requestedCapacity: Int!
-    pendingCapacity: Int!
-    runningCapacity: Int!
-    lastActive: DateTime!
-    lastResolved: DateTime!
-    pendingTasks: Int!
-    failed: Int!
-    exception: Int!
-    unscheduled: Int!
-    provider: String!
+    config: WMWorkerPoolGeneralConfig
```

Comment:
`config` is defined as an object in https://github.com/taskcluster/taskcluster/blob/master/services/worker-manager/schemas/v1/worker-pool-full.yml. Are we sure those properties won't change for a different worker pool? Is it possible we want to show a different table depending on the type of worker pool? Do we foresee a future where a user may want to see a certain value returned in config but this value happens not to be relevant for a different worker pool?

For property values that don't have a well defined structure, I think it makes sense to make the return value a JSON.

```jsx
config: JSON
```

Reply:
> Are we sure those properties won't change for a different worker pool? 

It's not used at the moment anyways, I will remove that property altogether

Reply:
> Are we sure those properties won't change for a different worker pool? Is it possible we want to show a different table depending on the type of worker pool? Do we foresee a future where a user may want to see a certain value returned in config but this value happens not to be relevant for a different worker pool?

But to clear this up: I believe there was a discussion  about this yesterday in the channel. I believe you will find the background information on this, as well as my opinion on this, there in that discussion. In short: yes, we do foresee and we are aware :)

Another thing to explain, you probably notice the word `general` in the name of the type. That was meant to be a type that has some fields that are common for all the providers.

But anyways, since it's not used anyways, I am just removing that type altogether